_includes/tutorials/aggregating-sum/kstreams/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,13 @@
+Use the `reduce()` method to apply the sum aggregation, see below.
+
+```
+    builder.stream(inputTopic, Consumed.with(Serdes.String(), ticketSaleSerde))
+        // Set key to title and value to ticket value
+        .map((k, v) -> new KeyValue<>((String) v.getTitle(), v.getTicketTotalValue()))
+        // Group by title
+        .groupByKey(Grouped.with(Serdes.String(), Serdes.Integer()))
+        // Apply SUM aggregation
+        .reduce(Integer::sum)
+        // Write to stream specified by outputTopic
+        .toStream().mapValues(v -> v.toString()).to(outputTopic, Produced.with(Serdes.String(), Serdes.String()));
```

Comment:
one final comment - in our short answer version just use `toStream().to(...)` but I'm on the fence about it.  Your call to make the change or not.

Reply:
Ah thank you for catching that. In commit https://github.com/confluentinc/kafka-tutorials/pull/814/commits/d5f0131fda1a9fc9e03f3ed643803734adefc08d I update the short answer with the " total sales" text.